### PR TITLE
Add details to setup instructions

### PR DESCRIPTION
**Pull request by m2-farzan**
This PR improves setup instructions in README. It could save people from some gotchas.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,48 @@
 # Mainflux IoT Admin UI based on Angular 8+ and <a href="https://github.com/akveo/nebular">Nebular</a>
 
-## Install
-Build Docker image:
+## Installation
```
Comment (drasko) [inline README.md:3]: It's `Install`. Stay consistent with other services.

File: README.md [hunk 2]
```diff
@@ -1,22 +1,48 @@
...
+
+### Quick Setup
```
Comment (drasko) [inline README.md:5]: No need for this subtitle.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,48 @@
...
+$ make run
 ```
+UI should be now up and running at `http://localhost/`.
 
-Run docker-compose:
+*(Note that `http://localhost:3000/` is for internal use only, and is not intended to be used by the end-user.)*
+
+More configuration (port numbers, etc.) can be done by editing the `.env` file before `$ make run`.
+
+### Developer Setup
```
Comment (drasko) [inline README.md:22]: Move this into `Usage` chapter, and just in one sentence explain that developer build can be achieved with this `make` commmand, but would be also goot to list a command that `make ui` executes.

File: README.md [hunk 4]
```diff
@@ -1,22 +1,48 @@
...
+For setting up a developer environment, Mainflux UI docker image must be built using the following command:
 ```
-make run
+$ git clone https://github.com/mainflux/ui.git
```
Comment (drasko) [inline README.md:25]: No need to repeat these steps, they are already handled in `Install`

**Review by drasko — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -19,14 +17,14 @@ UI should be now up and running at `http://localhost/`.
 
 More configuration (port numbers, etc.) can be done by editing the `.env` file before `$ make run`.
 
-### Developer Setup
-For setting up a developer environment, Mainflux UI docker image must be built using the following command:
+## Usage
+A developer build from the source can be achieved using the following command:
 ```
 $ git clone https://github.com/mainflux/ui.git
```
Comment (drasko) [inline README.md:23]: Please remove these lines if the repeat installation process - mentioning just what `make ui` brings up developer version is sufficient.

**Review by drasko — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (m2-farzan) [reply README.md]: I'm a bit confused here. I renamed the chapter to `usage` and trimmed the sentences but maybe I didn't understand you correctly.

Comment (drasko) [reply README.md]: This looks good.

File: README.md (hunk 5 above)
Comment (m2-farzan) [reply README.md]: Should I also remove the second `make run` and also `angular-cli` stuff?

**Review by manuio — APPROVED** [review]
LGTM

File: README.md [hunk 6]
```diff
@@ -1,22 +1,44 @@
...
 ## Install
-Build Docker image:
+For a quick setup, pre-built images from Docker Hub can be used.
+
+First, make sure that `docker` and `docker-compose` are installed. Also, stop existing Mainflux containers if any.
+
+Then, use the following instructions:
 ```
-make ui
+$ git clone https://github.com/mainflux/ui.git
```
Comment (drasko) [inline README.md:10]: No need for `$` sign, only makes copy-pasting hrder.

File: README.md [hunk 7]
```diff
@@ -1,22 +1,44 @@
 # Mainflux IoT Admin UI based on Angular 8+ and <a href="https://github.com/akveo/nebular">Nebular</a>
 
 ## Install
-Build Docker image:
+For a quick setup, pre-built images from Docker Hub can be used.
+
+First, make sure that `docker` and `docker-compose` are installed. Also, stop existing Mainflux containers if any.
+
+Then, use the following instructions:
 ```
```
Comment (drasko) [inline README.md:9]: Add `bash` after triple backtick

**Review by drasko — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Comment by manuio** [comment]
Thank you @m2-farzan !

**Pull request merged**